### DISCUSSION TITLE
how can I make the file look as it is in my profile?

### DISCUSSION THREAD

**Discussion by jackvvl** [lowlighter/metrics#1369] [Q&A]
hi, how can I make the file look as it is in my profile?  Please enlighten me. Srry if I wrote something nonsense, the google translator is sometimes wrong

![yes](https://user-images.githubusercontent.com/78714133/214424968-227cc7f4-e8c5-48d6-bb01-5511e934b268.jpg)

**Comment by lowlighter** [comment]
Hi 👋 !

I'm sorry, I'm not sure I understood 😅 
Are you asking how you can dow to make the image "fill the width" of your profile ?

**Reply by jackvvl** [reply]
Don't worry, I've already solved my problem 😁 thanks.
